project_parent_task_filter/readme/DESCRIPTION.rst

```diff
@@ -1,4 +1,4 @@
-This module adds a filter to show only the parent tasks in a project and
+This module adds a filter to show only the parent or non-parent tasks in a project and
```

Comment:
This module adds two filters to show only the parent or non-parent tasks in a project and